tests/backends/storage/test_fs.py

```diff
@@ -0,0 +1,105 @@
+"""Tests for Ralph fs storage backend"""
+
+from collections.abc import Iterable
+from pathlib import Path
+
+import pytest
+
+from ralph.backends.storage.fs import FSStorage
+from ralph.defaults import FS_STORAGE_DEFAULT_PATH
+
+
+def clean_fs_storage(path):
+    """Delete all content of the storage and its path"""
+    posix_path = Path(path)
+    for file in posix_path.iterdir():
+        file.unlink()
+    if path.endswith("/"):
+        path = path[:-1]
+    path = path.split("/")
+    while len(path) > 0:
+        new_path = "/".join(path)
+        Path(new_path).rmdir()
+        path.pop()
```

Comment:
Use pyfakefs instead (see examples using the `fs` fixture in other tests).

Reply:
It is now used in all tests that need it but one, thank you